version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.20.1'
```

Comment:
@felliott this will be the file that needs manual version bump for release/hotfix
@Johnetordoff need to update this to 0.22.0